protoc-gen-openapiv2/internal/genopenapi/types.go

```diff
@@ -257,16 +257,12 @@ type keyVal struct {
 type openapiSchemaObjectProperties []keyVal
 
 func (p openapiSchemaObjectProperties) MarshalYAML() (interface{}, error) {
-	ms := make(yaml.MapSlice, len(p))
-
-	for i, v := range p {
-		ms[i] = yaml.MapItem{
-			Key:   v.Key,
-			Value: v.Value,
-		}
+	yamlContent, err := yaml.Marshal(p)
```

Comment:
I think this is going to cause an infinite loop 😂

Reply:
Can't believe I missed that :facepalm: